hardhat.config.ts

```diff
@@ -9,7 +9,6 @@ import "./big-int-fix.ts";
 dotenv.config();
 
 // eslint-disable-next-line import/first
```

Comment:
This line is also not required anymore, because we don't have imports under the comment.